### PR TITLE
First pass at adding permission notes to features (DOCS-304)

### PR DESCRIPTION
**Pull request by djw-m**
## What Changed?

Adding notes where bdr_application role will be needed for a user,

So far, sequences, triggers and CAMO. May be more needed.

### CODE REVIEW

File: product_docs/docs/pgd/5/consistency/column-level-conflicts.mdx [hunk 1]
```diff
@@ -35,6 +35,10 @@ Column-level conflict resolution requires the table to have `REPLICA IDENTITY FU
 
 ## Enabling and disabling column-level conflict resolution
 
+!!! Note Permissions Required
+Column level conflict detection uses the `column_timestamp` type. This requires the user to have have at least the [bdr_application](security/pgd-predefined-roles/#bdr_application) role assigned to them.
```
Comment (pgnickb) [inline product_docs/docs/pgd/5/consistency/column-level-conflicts.mdx:39]:
```suggestion
Column level conflict detection uses the `column_timestamps` type. This requires the user to have have at least the [bdr_application](security/pgd-predefined-roles/#bdr_application) role assigned to them.
```

File: product_docs/docs/pgd/5/consistency/crdt.mdx [hunk 2]
```diff
@@ -7,6 +7,10 @@ redirects:
 
 Conflict-free replicated data types (CRDT) support merging values from concurrently modified rows instead of discarding one of the rows as traditional resolution does.
 
+!!! Note Permissions Required
+PGD CRDT require execute access to CRDT types, therefore a user must have at least the [bdr_application](security/pgd-predefined-roles/#bdr_application) role assigned to them.
```
Comment (pgnickb) [inline product_docs/docs/pgd/5/consistency/crdt.mdx:11]:
```suggestion
PGD CRDT require usage access to CRDT types, therefore a user must have at least the [bdr_application](security/pgd-predefined-roles/#bdr_application) role assigned to them.
```

File: product_docs/docs/pgd/5/sequences.mdx [hunk 3]
```diff
@@ -18,6 +18,12 @@ produce values that are unique only on the local node. This is important because
 unique ids generated by such sequences cause conflict and data loss by
 means of discarded `INSERT` actions in multi-master replication.
 
+!!! Note Permissions Required
+PGD global sequences are on by default. This means that for any user to use sequences,
```
Comment (pgnickb) [inline product_docs/docs/pgd/5/sequences.mdx:22]: Do we have docs that discuss how/when normal sequences are converted to global ones? Perhaps worth linking it here?

**Review by ibarwick — APPROVED** [review]
LGTM

File: product_docs/docs/pgd/5/reference/functions-internal.mdx [hunk 4]
```diff
@@ -367,17 +367,23 @@ Currently, row_filters are ignored by this function.
...
 
-### `bdr.seq_nextval`
+### `bdr.seq_currval`
 
-Internal implementation of sequence increments.
+Part of the internal implementation of global sequence manipulation.
 
-Use this function instead of standard `nextval` in queries that
-interact with [PGD global sequences](../sequences/#pgd-global-sequences).
+Invoked automatically as needed.
```
Comment (djw-m) [inline product_docs/docs/pgd/5/reference/functions-internal.mdx:374]:
```suggestion
Invoked automatically when `currval()` is called on a galloc or snowflakeid sequence.
```

File: product_docs/docs/pgd/5/reference/functions-internal.mdx [hunk 5]
```diff
@@ -367,17 +367,23 @@ Currently, row_filters are ignored by this function.
...
 
-#### Notes
+### `bdr.seq_lastval`
+
+Part of the internal implementation of global sequence manipulation.
+
+Invoked automatically as needed.
```
Comment (djw-m) [inline product_docs/docs/pgd/5/reference/functions-internal.mdx:380]:
```suggestion
Invoked automatically when `lastval()` is called on a galloc or snowflakeid sequence
```

File: product_docs/docs/pgd/5/reference/functions-internal.mdx [hunk 6]
```diff
@@ -367,17 +367,23 @@ Currently, row_filters are ignored by this function.
...
+
+### `bdr.seq_nextval`
+
+Part of the internal implementation of global sequence increments.
 
-The following are also internal PGD sequence manipulation functions.
-`bdr.seq_currval` and `bdr.seq_lastval` are used automatically.
+Invoked automatically as needed.
```
Comment (djw-m) [inline product_docs/docs/pgd/5/reference/functions-internal.mdx:386]:
```suggestion
Invoked automatically when `nextval()` is called on a galloc or snowflakeid sequence
```

**Review by pgnickb — APPROVED** [review]

**Pull request merged**